### PR TITLE
allow for specifying setup job resources

### PR DESCRIPTION
**Pull request by DavidSpek**
This PR allows for setting the resources on the setup job. Currently, the requested resources are quite high, in particular for the elasticsearch setup job that just runs curl, which can cause unnecessary cluster scale ups. This PR makes is possible to specify the resources for the various setup jobs so people can configure these according too their needs.

## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

File: charts/datahub/templates/elasticsearch-setup-job.yml [hunk 1]
```diff
@@ -89,13 +89,10 @@ spec:
...
           {{- end }}
+          {{- toYaml .Values.elasticsearchSetupJob.resources | nindent 8 }}
           resources:
-            limits:
-              cpu: 500m
-              memory: 512Mi
-            requests:
-              cpu: 300m
-              memory: 256Mi
+            {{ . }| nindent 12 }}
```
Comment (szalai1) [inline charts/datahub/templates/elasticsearch-setup-job.yml:94]: are you sure this `}|` is going to work?

**Comment by szalai1** [comment]
hey @DavidSpek, thanks for the contribution. Looks good. Left one comment.

**Comment by DavidSpek** [comment]
Thanks for the review. I've fixed the typo so it should be good now.

**Comment by DavidSpek** [comment]
I've just fixed the merge conflict.

**Review by szalai1 — COMMENTED** [review]
@DavidSpek Please fix the conflict, but it's good to be merged

File: charts/datahub/Chart.yaml [hunk 2]
```diff
@@ -4,7 +4,7 @@ description: A Helm chart for LinkedIn DataHub
 type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.2.108
+version: 0.2.109
```
Comment (szalai1) [inline charts/datahub/Chart.yaml:7]:
```suggestion
version: 0.2.111
```

**Comment by pedro93** [comment]
Hello @DavidSpek 

Could you resolve the chart.yaml conflict? With that done we can merge this in :)

**Comment by pedro93** [comment]
Closing PR in favor of https://github.com/acryldata/datahub-helm/pull/224

**Pull request closed without merging**